src/ui/utils/tokenManager.tsx

```diff
@@ -253,6 +254,11 @@ class TokenManager {
     );
     this.refreshTimeout = window.setTimeout(() => this.update(true), refreshDelay);
   }
+
+  componentWillUnmount() {
+    clearTimeout(this.refreshTimeout);
+    clearTimeout(TokenManager.timeOutID);
```

Comment:
This looks like another troublesome mix of static property (`TokenManager.timeOutID`) and _component instance_ timeouts.

Reply:
Thank you for your comments and notes @bvaughn. Actually these changes are auto-generated by our leak-fixing tool that we are working on. Our tool is not yet smart enough to detect whether the timers are associated with the class or instance (hence the static props). If you'd like, I can manually fix up these issues as per your comments. :)

Reply:
Ah, that's interesting!

No, you don't have fix them. We could certainly do it. I just wanted to note them so we didn't merge them as-is.